src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6325,6 +6325,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_PartialMethodRefReturnDifference" xml:space="preserve">
     <value>Partial method declarations must have matching ref return values.</value>
   </data>
+  <data name="WRN_PartialMethodTypeDifference" xml:space="preserve">
+    <value>Partial method declarations '{0}' and '{1}' have signature differences.</value>
+  </data>
+  <data name="WRN_PartialMethodTypeDifference_Title" xml:space="preserve">
+    <value>Partial method declarations have differences in parameter names, parameter types, or return types.</value>
```

Comment:
Ah, hate to do this last minute, but this message should be changed to be in sync with the other message.

Reply:
@RikkiGibson My bad! Fixing now.